README.md

```diff
@@ -369,6 +369,21 @@ df = results_to_dataframe(results)
 
 </details>
 
+<details>
+  <summary>  Caching Embeddings To Re-Use Them </summary>
+
+
+### Caching Embeddings To Re-Use Them
+
+There are times you may want to cache the embeddings so you can re-use them. This may be true if you have multiple query sets for the same corpus (e.g. Wikipedia) or are doing some optimization over the queries (e.g. prompting, other experiments). You can setup a cache by using a simple wrapper, which will save the cache per task in the `cache_embeddings/{task_name}` folder:
+
+```python
+... # define your task and model above as normal
+evaluation.run(model, encode_kwargs={'cache_embeddings': 'path_to_cache_dir', "vector_dim": 512}) # setting this flag will create/use the cache
```

Comment:
Maybe make this 
```python
evaluation.run(model, cache_embeddings='path_to_cache_dir', vector_dim= 512)
```
Also, is it possible to make it without specifying vector_dim?

Reply:
+1, you can not set it it and it will default to 768.  But to create the mmap matrix it needs some dimension.

I could have it wait until it encodes something and then get the dimension/create the matrix? And/or store a dimensionality number in the cache so it could be loaded without it needing to be passed. Does that seem preferable?

Reply:
I see that you passed default, but what will be if model has 1024 vector size? Will it be loaded correctly? I think that load from cache will be more preferable

Reply:
Sounds good to me, added the inferring details in so it doesn't have to be passed.